api/client/login.go

```diff
@@ -22,7 +23,7 @@ import (
 //
 // Usage: docker login SERVER
 func (cli *DockerCli) CmdLogin(args ...string) error {
-	cmd := Cli.Subcmd("login", []string{"[SERVER]"}, "Register or log in to a Docker registry server, if no server is\nspecified \""+registry.IndexServer+"\" is the default.", true)
+	cmd := Cli.Subcmd("login", []string{"[SERVER]"}, api.DockerCommands["login"].Description+"\nif no server is specified \""+registry.IndexServer+"\" is the default.", true)
```

Comment:
If we're going to use this trick of using cmd.Description as the first part of a longer description then I think we need to assume that cmd.Description is a complete sentence and that any extra text starts a new sentence on a new line. Assuming that the text in cmd.Description will always flow/read correctly with text kept here is very risk.
I like what you did at: https://github.com/docker/docker/pull/16674/files#diff-c943834b9f6589e94e6f3d2df975fd45R48
So here I would make it:
```
... api.DockerCommands["login"].Description+"\nIf no server is specified....
```
Basically make "if" start a new sentence.